spec/actions_specs/crashlytics_spec.rb

```diff
@@ -212,15 +212,16 @@
               end").runner.execute(:test)
 
             expect(command).to eq([
-              @crashlytics_bundle,
-              "wadus",
-              "secret",
-              "-ipaPath './fastlane/spec/fixtures/fastfiles/Fastfile1'",
-              "-emails 'email1,email2'",
-              "-notesPath './fastlane/spec/fixtures/fastfiles/Fastfile1'",
-              "-groupAliases 'groups,123'",
-              "-notifications NO",
-              "-debug NO"])
+                                    @crashlytics_bundle,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.